ribs/archives/_archive_base.py

```diff
@@ -4,7 +4,6 @@
 
```

Comment:
Make sure you rename `get_index` to `index_of` since it is an abstract method in this base class